moler/connection_observer.py

```diff
@@ -334,6 +337,14 @@ def extend_timeout(self, timedelta):  # TODO: probably API to remove since we ha
         self.runner.timeout_change(timedelta)
         self._log(logging.INFO, msg)
 
+    def on_inactivity(self):
+        """
+        Callback called when in self.inactivity_timeout seconds no data are sent by connection.
```

Comment:
Callback called when no data is received on connection within self.inactivity_timeout seconds

Reply:
Done.